### ISSUE TITLE
Basic melee attack

### ISSUE THREAD

**Issue by Kontur86911** [praydog/UEVR#164]
It would be great to see some basic melee atacks when you move your motion controller fast enough. It was done in Vivecraft and many other mods. 
Just simple fast move of the controller in space = pressed attack button. Needs to be an option in input section

**Comment by praydog** [comment]
I think there is generic damage code within UE (actor damage or something?), but not sure how many games will make use of this. I will have to sit down and research this unless someone already knows the answer to this

**Comment by Kontur86911** [comment]
> But a gesture to just press a button would be easier

Yes. Most of the mods are made pretty simple. At the moment you have to press the trigger while swinging your melee weapon. If there is an option not to press the button, it would feel much more immersive.
Also there are games with guns, where close combat attack is binded to R3 button, so it would be great to see 2 buttons there for choice. 
Swing = press trigger. And swing = press R3 button.
So even if you're carrying a gun, you'll still be able to perform melee attacks just with a swing of your arm, leaving the trigger for shooting.
